Broadcom/siteminder/tests/AuthReject.json

```
@@ -0,0 +1,68 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Broadcom Siteminder",
+        "dialect_uuid": "6a740c4b-468b-45b4-9982-3903abf9fc54"
+      }
+    },
+    "message": "AuthReject HOST_XXX [23/Jul/2025:09:36:18 +0200] \"3.3.3.3 tata\" \"domain.com GET /affwebservices/redirectjsp/redirect_unique.jsp?SAMLRequest=fZJRS8MwFIX%2FSsl7mrR1nQtdYWwIBRVR8cG3NL3dAm1Sc1On%2F960RUEGe73ck3O%2Bc1Og7LtB7EZ%2FMs%2FwMQL6qDpsiW5onq%2FyRGYp5Snn9IbzDd20m5Q2G6hXdZ4quc5I9AYOtTVbksacRBXiCJVBL40PI56uKF%2FTNHvla5Hlgt%2FGeZ68k%2BgQfLSRflaevB9QMIZoqW6G2J9kB3h0dhxiZXsm2%2FYMNYL71AqQDWPdacWm4GmQkGhvDcLkNzojrESNwsgeUHglXnYP9yJEE2pZEqPBAZRuNTQkurNOwcy%2BJd6NQKKvvjMo5lKuvzc4662yHSmLGdot0usiiYFigiblL3Rnj9rEgeMSu2mR%2FVGHeOgLtniVxXK2x%2BBQHZ5s6ON7gunllRaSOJkn4bTtvPq%2Fi13X2fPegfTwWwYrC3b5P8of&RelayState=c9fc2739-6a5b-436a-90e4-aca86ee9a636&SMPORTALURL=https%3A%2F%2Fsso-idp.xxxxgroup.com%2Faffwebservices%2Fpublic%2Fsaml2sso&SAMLTRANSACTIONID=529c8b77-146ba9a3-74ecf707-0641d84a-9afd1ab\" [] [0] Invalid credentials [] []"
+  },
+  "expected": {
+    "message": "AuthReject HOST_XXX [23/Jul/2025:09:36:18 +0200] \"3.3.3.3 tata\" \"domain.com GET /affwebservices/redirectjsp/redirect_unique.jsp?SAMLRequest=fZJRS8MwFIX%2FSsl7mrR1nQtdYWwIBRVR8cG3NL3dAm1Sc1On%2F960RUEGe73ck3O%2Bc1Og7LtB7EZ%2FMs%2FwMQL6qDpsiW5onq%2FyRGYp5Snn9IbzDd20m5Q2G6hXdZ4quc5I9AYOtTVbksacRBXiCJVBL40PI56uKF%2FTNHvla5Hlgt%2FGeZ68k%2BgQfLSRflaevB9QMIZoqW6G2J9kB3h0dhxiZXsm2%2FYMNYL71AqQDWPdacWm4GmQkGhvDcLkNzojrESNwsgeUHglXnYP9yJEE2pZEqPBAZRuNTQkurNOwcy%2BJd6NQKKvvjMo5lKuvzc4662yHSmLGdot0usiiYFigiblL3Rnj9rEgeMSu2mR%2FVGHeOgLtniVxXK2x%2BBQHZ5s6ON7gunllRaSOJkn4bTtvPq%2Fi13X2fPegfTwWwYrC3b5P8of&RelayState=c9fc2739-6a5b-436a-90e4-aca86ee9a636&SMPORTALURL=https%3A%2F%2Fsso-idp.xxxxgroup.com%2Faffwebservices%2Fpublic%2Fsaml2sso&SAMLTRANSACTIONID=529c8b77-146ba9a3-74ecf707-0641d84a-9afd1ab\" [] [0] Invalid credentials [] []",
+    "event": {
+      "action": "AuthReject",
+      "category": [
+        "authentication"
+      ],
+      "dataset": "smaccess",
+      "kind": "event",
+      "message": "Invalid credentials ",
+      "outcome": "failure",
+      "reason": "Sm_Api_Reason_None",
+      "type": [
+        "denied"
```

"denied" is not a valid type for authentication events (see [spec](https://www.elastic.co/docs/reference/ecs/ecs-allowed-values-event-category#ecs-event-category-authentication)).
According to the event, `start` is the expected type with the outcome `failure`
```suggestion
        "start"
```

---

For me, the issue here is that start is not present, i agree with you. 
For the denied type, I removed it after reviewing my test logs as reject better translates to failure in these logs. 

But as I understand the ECS standard, expected doesn't mean "authorized only" so a event.type ["start","denied"] would be correct. 

Do you agree ?

For this particular integration, I removed the denied type.

Thanks for your review